packages/hotjar/src/hotjar.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-console */
+
```

Comment:
It might be a good idea to drop a link to Github where we sourced/adapted this from for future reference.